### PR TITLE
Add percentage next to the active, recovered and deceased titles

### PR DESCRIPTION
**Pull request by jagadeshanh**
**Description of PR**

Added percentage to title tags (Active, Recovered, Deceased)

**Checklist**

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

**Screenshots**

Screen Shot 2020-06-17 at 6 54 54 PM

### CODE REVIEW

File: src/components/level.js [hunk 1]
```diff
@@ -21,7 +26,13 @@ function PureLevelItem({statistic, total, delta}) {
 
   return (
     <React.Fragment>
-      <h5>{t(capitalize(statistic))}</h5>
+      <h5>
+        {t(capitalize(statistic)) +
+          ' ' +
+          (statistic !== 'confirmed'
+            ? getPercentageText(confirmed, total)
+            : '')}
```
Comment (ShaileshPrajapati-BTC) [inline src/components/level.js:34]: @jagadeshanh  don't need to write ternary operator for this
`(statistic !== 'confirmed' && getPercentageText(confirmed, total))`

File: src/utils/commonfunctions.js [hunk 2]
```diff
@@ -72,6 +72,11 @@ export const capitalize = (s) => {
   return s.charAt(0).toUpperCase() + s.slice(1);
 };
 
+export const getPercentageText = (confirmed, total) => {
+  const percentage = (total * 100) / confirmed;
+  return '(' + Math.round(percentage * 100) / 100 + '%' + ')';
```
Comment (ShaileshPrajapati-BTC) [inline src/utils/commonfunctions.js:77]: @jagadeshanh  always use the string interpolation for this kind of manipulation  
``` 
return `( ${Math.round(percentage * 100) / 100} % )`
```

File: src/components/level.js (hunk 1 above)
Comment (jagadeshanh) [reply src/components/level.js:34]: Thank you I will update the PR.

**Comment by shuklaayush** [comment]
Please read the [Contributing Guidelines](https://github.com/covid19india/covid19india-react/blob/master/CONTRIBUTING.md) before creating a PR

**Pull request closed without merging**